ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -120,24 +120,13 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
     }
 
     /// Set `backgroundColor` to customize background color of the drawer
-    @objc open var backgroundColor: UIColor {
-        get {
-            let color: DynamicColor
-            if presentationController is UIPopoverPresentationController {
-                color = tokenSet[.popoverContentBackground].dynamicColor
-            } else if useNavigationBarBackgroundColor {
-                color = tokenSet[.navigationBarBackground].dynamicColor
-            } else {
-                color = tokenSet[.drawerContentBackground].dynamicColor
-            }
-            return UIColor(dynamicColor: color)
-        }
-        set {
+    @objc lazy open var backgroundColor: UIColor = DrawerController.drawerBackground(fluentTheme: view.fluentTheme) {
```

Comment:
Since this is lazy, we should be able to access the tokenSet here. We should be able to do
```
@objc lazy open var backgroundColor = UIColor(dynamicColor: tokenSet[.drawerContentBackground].dynamicColor) {
```